src/features/ModelPlan/CollaborationArea/Cards/cards.scss

```diff
@@ -39,4 +39,15 @@
       }
     }
   }
+
+  &__main-card {
+    min-height: 420px;
+  }
```

Comment:
Need some responsive styles to address links/button stacking on top of another when screen is smaller

![image](https://github.com/user-attachments/assets/9aa7d4ab-e374-4c71-b854-8dea84c880ff)